### ISSUE TITLE
Notification mail was not sent when deleting a managed user

### ISSUE THREAD

**Issue by TakeZNt** [box/box-java-sdk#774]
- [x] I have checked that the [SDK documentation][sdk-docs] and [API documentation][api-docs] doesn't solve my issue

### Description of the Issue
I deleted a managed user by BoxUser#delete method.  I set the first argument (notifyUser) true.
I expected a notification mail is sent, but was not. What's wrong with me?

### Versions Used
Java SDK: 2.44.0
Java: 1.8.0_181

### Steps to Reproduce
1. Create a managed user.
1. Delete the user by BoxUser.delete method with the first argument true.
1. No notification mail is sent.

### Error Message, Including Stack Trace
Deletion was succeeded, so no error messages.

**Comment by sujaygarlanka** [comment]
@TakeZNt Could you share the code you used to delete the BoxUser?

**Comment by TakeZNt** [comment]
Thanks for reply. This is the code snipet.

```
public void deleteUser(final String userId, final boolean notify, final String transferToId) throws BoxClientException {
	try {
		final BoxUser user = new BoxUser(connection.getOriginalConnection(), userId);
		if (transferToId != null) {
			user.transferContent(transferToId);
		}
		// force is always true
		user.delete(notify, true);
	} catch (BoxAPIException e) {
		throw new BoxClientException(e);
	}
}
```

I call the `deleteUser` with notify = true.

**Comment by sujaygarlanka** [comment]
@TakeZNt This is an issue with the backend service that sends notification emails. This is a result of recently migrating our notification services to a new service. We currently have a ticket open to fix this issue. We will keep you up to date on any progress.

**Comment by sujaygarlanka** [comment]
@TakeZNt Is the issue still occurring?

**Comment by TakeZNt** [comment]
I tried again right now, and Yes, the issue still occurs.

**Comment by antusus** [comment]
Hello @TakeZNt ,

I was able to reproduce this issue. I tried that with UI and I also didn't get any email when removing user. If this urgent I can suggest opening a [support ticket](https://support.box.com/hc/en-us/requests/new) which should get you in contact with notifications team and work with them on solving the issue.

I'm closing this issue as it is not related to SDK and we cannot help any further.
